batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/utils/DbConnection.java

```diff
@@ -0,0 +1,40 @@
+package com.lftechnology.batch7crud.utils;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @Author binodnme
+ * Created on 1/18/16
+ */
+public class DbConnection {
+
+    private static Logger logger = Logger.getLogger(DbConnection.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)